db/seeds.rb

```diff
@@ -5,3 +5,38 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+User.destroy_all
```

Comment:
@Gataniel why it's all here? I think we already told about it that there should not be test data in seeds.